docs/installation/_index.md

```diff
@@ -82,9 +72,28 @@ Proceed to [install `viam-micro-server`](/installation/#install-viam-micro-serve
 {{% /tab %}}
 {{< /tabs >}}
 
+### Operating System installation
+
+`viam-server` can run on **any SBC that runs a [supported operating system](#platform-requirements)**.
+
+For convenience, we provide operating system installation instructions for some popular boards.
```

Comment:
```suggestion
For convenience, we provide operating system installation instructions for some popular SBCs.
```

Reply:
could fit with the definition of sbcs above or could define here

Reply:
yeah nice